src/screens/Accounts/index.js

```diff
@@ -64,6 +92,34 @@ class Accounts extends NavigatorComponent<Props & Actions & AccountsState> {
     });
   };
 
+  static onForgetPasswordAccount = async (navigator: Navigator, currentAccountId: string) => {
+    if (Platform.OS === 'ios') {
+      await navigator.dismissModal();
+      navigator.push({
```

Comment:
`navigator.push(...)` line is the same for both platforms, so can be moved outside of the `if` statement.

Reply:
done